src/main/java/com/lorepo/icplayer/client/module/text/DraggableGapWidget.java

```diff
@@ -47,7 +48,7 @@ public DraggableGapWidget(GapInfo gi, final ITextViewListener listener) {
 				public void onClick(ClickEvent event) {
 					event.stopPropagation();
 					event.preventDefault();
-					if (listener != null && !disabled && isWorkMode) {
+					if (listener != null && isWorkMode && _disabled) {
```

Comment:
To _disabled jest odwrócone. 
Można do wyłączonej gapy włożyć element (przez kliknięcie).